recipe/meta.yaml

```diff
@@ -24,6 +24,8 @@ requirements:
     - requests
     - requests_cache
     - webob
+    - defusedxml
+    - myproxyclient
 
 test:
   imports:
```

Comment:
Maybe you could add an import of `pyesgf.logon` as a test here?

Reply:
done